main/src/main/java/org/mobilitydata/gtfsvalidator/notice/StopLocationWithoutParentStationNotice.java

```diff
@@ -26,8 +26,8 @@
  *
  * <p>Severity: {@code SeverityLevel.ERROR}
  */
-public class LocationWithoutParentStationNotice extends ValidationNotice {
-  public LocationWithoutParentStationNotice(
+public class StopLocationWithoutParentStationNotice extends ValidationNotice {
```

Comment:
What is the rationale for that change? This notice applies to stops, entrances and boarding areas.

Reply:
@barbeau  suggested in https://github.com/MobilityData/gtfs-validator/pull/781#discussion_r585866237 we 

> put the entity name first when it makes sense for consistency, and because if we list the rules in the table alphabetically then, for example, most of the Route rules should be grouped together. 

This change would allow to regroup notices related to `stops.txt`